react/features/conference/components/DisplayNameLabel.native.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import {
+    getLocalParticipant,
+    getParticipantDisplayName,
+    shouldRenderParticipantVideo
+} from '../../base/participants';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * The name of the participant to render.
+     */
+    _participantName: string,
+
+    /**
+     * True of the label needs to be rendered. False otherwise.
+     */
+    _render: boolean
+}
+
+/**
+ * Renders a label with the display name of the on-stage participant.
+ */
+class DisplayNameLabel extends Component<Props> {
+    /**
+     * Implements {@code Component#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        if (!this.props._render) {
+            return null;
+        }
+
+        return (
+            <View style = { styles.displayNameLabelWrapper }>
```

Comment:
Can we make it work with a single wrapper view and combine those 2 styles?

Reply:
I wasn't sure that's possible with the surrounding layout, but yeah, managed to do it finally. Thanks!